test/acceptance/customresourcedefinition_oapi3_test.go

```diff
@@ -0,0 +1,176 @@
+// +build acceptance
+
+package acceptance
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/terraform-provider-kubernetes-alpha/provider"
+	tfstatehelper "github.com/hashicorp/terraform-provider-kubernetes-alpha/test/helper/state"
+)
+
+func TestKubernetesManifest_CustomResource_OAPIv3(t *testing.T) {
+	kind := strings.Title(randString(8))
+	plural := strings.ToLower(kind) + "s"
+	group := "terraform.io"
+	version := "v1"
+	groupVersion := group + "/" + version
+	crd := fmt.Sprintf("%s.%s", plural, group)
+
+	name := strings.ToLower(randName())
+	namespace := "default" //randName()
+
+	tfvars := TFVARS{
+		"name":          name,
+		"namespace":     namespace,
+		"kind":          kind,
+		"plural":        plural,
+		"group":         group,
+		"group_version": groupVersion,
+		"cr_version":    version,
+	}
+
+	step1 := tfhelper.RequireNewWorkingDir(t)
+	step1.SetReattachInfo(reattachInfo)
+	defer func() {
+		// step1.RequireDestroy(t)
```

Comment:
That's something I forgot there from my manual testing.
It was meant so it would leave the resources on the cluster, so I can inspect them manually. 

Thanks for catching it.